dnf-behave-tests/fixtures/specs/dnf-ci-rich/porridge-1.0-1.spec

```diff
@@ -8,7 +8,7 @@ URL:            None
 
 Summary:        The made up package for rich dependencies testing.
 
-Requires:       (milk if (float or oat))
+Requires:       (milk if (flour or oat))
```

Comment:
I guess that the change is not required by new scenarios. But it looks like a typo in require, but I am not sure. Otherwise it looks good.

Reply:
Yes, it is not required, but the mistake irritated me :smiley: Most probably a typo, because in tests comments it appears correctly using the `flour`.

Reply:
May be moving it into another commit would be better. I am curious why the change did not break any other test.